proto/api.proto

```diff
@@ -1908,6 +2056,93 @@ message GetOpenPerpOrderResponse {
   PerpOrder order = 2;
 }
 
+message GetDriftOpenMarginOrderRequest {
+  string ownerAddress = 1;
+  string accountAddress = 2;
+  uint64 orderID = 3;
+  uint64 clientOrderID = 4;
+}
+
+message GetDriftOpenMarginOrderResponse {
+  string ownerAddress = 1;
+  DriftMarginOrder order = 2;
+}
+
+message GetDriftOpenPerpOrderRequest {
```

Comment:
Do we still need the above message definition GetOpenPerpOrderRequest ? that have similar name and structure with exception of Project property. is this V1 vs V2 thing ?

Reply:
yeah, in V2, we don't use project property. since its clear it's a drift message